packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -191,6 +201,18 @@ export class EmrCreateCluster extends sfn.TaskStateBase {
     }
 
     this.taskPolicies = this.createPolicyStatements(this._serviceRole, this._clusterRole, this._autoScalingRole);
+
+    if (this.props.stepConcurrencyLevel !== undefined) {
+      if (this.props.stepConcurrencyLevel < 1 || this.props.stepConcurrencyLevel > 256) {
+        throw new Error(`Step concurrency level must be in range [1, 256], but got ${this.props.stepConcurrencyLevel}.`);
+      }
+      if (this.props.releaseLabel && this.props.stepConcurrencyLevel !== 1) {
+        const [major, minor] = this.props.releaseLabel.substr(4).split('.');
```

Comment:
This feels fragile -- what if the major version of EMR exceeds one digit?

Reply:
I think we are fine if the major version exceeds one digit. `releaseLabel` should look something like this `'emr-5.28.0'`. So in a perfect world, `emr-` gets removed by `releaseLabel.substr(4)` and then after splitting on `('.')`, we get `[5, 28, 0]`. Which won't have any issues if the major version exceeds one digit. 

_However_, there being no validation on `releaseLabel` (and it just being a string) means that one could simply have `releaseLabel = 'lalallalala'` and we'd instantly have a weird error since `this.props.releaseLabel.substr(4).split('.')` will not return what we expect. In this sense, I agree that it's fragile.

I'll add a `validateReleaseLabel()` function (which we should have to begin with). I think this solves the problem.

Reply:
@BenChaimberg thanks for the review from the other side :). Wondering if you have any nits for my new `validateReleaseLabel()`, which turns out not to be the simplest function ever.

Reply:
ah yea I misunderstood the purpose of `substr` (thought the release label looked like `'5.28.0'` and so `releaseLabel.substr(4)` would be `'5.28'`)